keps/sig-node/2400-node-swap/README.md

```diff
@@ -666,6 +839,22 @@ For beta 1:
 - Add e2e tests that verify pod-level control of swap utilization.
 - Add e2e tests that verify swap performance with pods using a tmpfs.
 
+For beta 2:
+
+- Add Node-conformance tests for basic swap validation. To avoid disrupting node conformance lanes, only the
+cgroup knobs are validated to be defined as expected with no real memory stress or swap use.
+
+For beta 3:
+
+- We want e2e tests that can confirm that eviction will take in account swap usage
+
+For GA:
+
+- Add a lane dedicated for swap testing, including stress tests and other tests that might be disruptive and intensive.
```

Comment:
FWIW I think it'd be safest to try to get this in beta 3 so we can catch potential issues as early as possible. I'd hate to have a delay to GA because we missed something a test exposed